python/graphstorm/gconstruct/remap_result.py

```diff
@@ -0,0 +1,477 @@
+"""
+    Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Remapping GraphStorm outputs (edge prediction results
+    node prediction results and node embeddings)
+    into the original node ID space.
+"""
+
+import os
+import argparse
+import logging
+import json
+import time
+import queue
+import math
+
+import torch as th
+from torch import multiprocessing
+from torch.multiprocessing import Process
+from ..model.utils import pad_file_index
+from .file_io import write_data_parquet
+from .id_map import IdReverseMap
+from ..utils import get_log_level
+
+from ..config import (GSConfig,
+                      get_argument_parser,
+                      BUILTIN_TASK_EDGE_CLASSIFICATION,
+                      BUILTIN_TASK_EDGE_REGRESSION,
+                      BUILTIN_TASK_NODE_CLASSIFICATION,
+                      BUILTIN_TASK_NODE_REGRESSION)
+
+id_maps = {}
+
+def worker_remap_edge_pred(pred_file_path, src_nid_path,
+    dst_nid_path, src_type, dst_type,
+    output_fname_prefix, chunk_size, preserve_input):
+    """ Do one edge remapping task
+
+        Parameters
+        ----------
+        pred_file_path: str
+            The path to the prediction result.
+        src_nid_path: str
+            The path to the file storing source node ids
+        dst_nid_path: str
+            The path to the file storing destination node ids
```

Comment:
should we store them in separate files?

Reply:
We already store them in separate files. src_nids_xxx and dst_nids_xxx.